build.sbt

```diff
@@ -23,7 +23,7 @@ name := "graphframes"
 spName := "graphframes/graphframes"
 
 // Don't forget to set the version
-version := s"0.6.0-SNAPSHOT-spark$sparkBranch"
+version := s"0.6.0-spark$sparkBranch"
 
```

Comment:
should we bump `spark.version` in line 4 above?

Reply:
I think 2.3.1 is the latest public version. I don't think we can claim 2.4 support until it's officially released.